Ports/openttd-opengfx/package.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=openttd-opengfx
+version=0.6.1
+workdir=.
+files="https://cdn.openttd.org/opengfx-releases/${version}/opengfx-${version}-all.zip opengfx-${version}-all.zip c694a112cd508d9c8fdad1b92bde05e7c48b14d66bad0c3999e443367437e37e"
+auth_type=sha256
+
+build() {
+    tar xf opengfx-${version}.tar
```

Comment:
wait why is this necessary? `files` are extracted automatically.

Reply:
Well, they have a `.tar` archive in a `.zip` file and our script only handles the outer `.zip` file. :man_facepalming: